### PR TITLE
Allow variables to be passed in at execution time, not only runtime (#37)

### PR DESCRIPTION
**Pull request by charles-dyfis-net**

### CODE REVIEW

File: src/jq/api.clj [hunk 1]
```diff
@@ -33,9 +33,9 @@
   ([^String query opts]
    (let [^JsonQuery json-query (impl/compile-query query)
          ^Scope scope (impl/new-scope opts)]
-     (fn ^String [^String data]
+     (fn ^String [^String data & {:keys [vars]}]
```
Comment (dainiusjocas) [inline src/jq/api.clj:36]: to keep it consistent I'd prefer to make this function with two arguments, first for data, second for options map.

Comment (charles-dyfis-net) [reply src/jq/api.clj]: Updated accordingly in 3dcaedfd5e5fd5bda78008ee7767ebb45028a9f4

Comment (charles-dyfis-net) [reply src/jq/api.clj]: (that committish is no longer in history as a separate item, but its contents have been folded in when rebasing/squashing this PR onto current main).

**Review by dainiusjocas — APPROVED** [review]
Nice, thanks for your contribution!

**Pull request merged**